src/customJs/tools/dynamic_tool/ProductViewer.tsx

```diff
@@ -131,6 +142,10 @@ export const ProductViewer: ViewerComponent<ProductValues> = ({
       href: '[[[DC:URL]]]',
       style: buttonStyle,
     },
+    info: {
```

Comment:
I see that it changed the description by the info attribute it is ok?

Reply:
Yes, it will not display product description, only info of the product like time since last visit, etc.